chia/wallet/trade_manager.py

```diff
@@ -481,17 +499,17 @@ async def respond_to_offer(self, offer: Offer, fee=uint64(0)) -> Tuple[bool, Opt
             confirmed=False,
             sent=uint32(0),
             spend_bundle=final_spend_bundle,
-            additions=final_spend_bundle.additions(),
-            removals=final_spend_bundle.removals(),
+            additions=[],
+            removals=[],
             wallet_id=uint32(0),
             sent_to=[],
-            trade_id=complete_offer.name(),
+            trade_id=bytes32([1] * 32),
```

Comment:
why do we set this to 1s? Will it overwrite the previous trade with trade id = 1

Reply:
im going to go ahead and merge so zlatko can get in the ui but will make sure this is revisted

Reply:
This is a dummy transaction record although you can't see the comment in the diff.  It's just for pushing the spend bundle, all of the actual transaction history is created separately.